tool/site/client/src/components/claim/createDispenseClaim.ts

```diff
@@ -287,29 +291,67 @@ function createEndorsementCodeableConcept(endorsement: EndorsementFormValues): f
   return endorsementCodeableConcept
 }
 
-const tempPractitionerRole: fhir.PractitionerRole = {
+const containedOrganization: fhir.Organization = {
+  resourceType: "Organization",
+  id: "organization",
+  identifier: [
+    {
+      system: "https://fhir.nhs.uk/Id/ods-organization-code",
+      value: "VNE51"
+    }
+  ],
+  address: [
+    {
+      city: "West Yorkshire",
+      use: "work",
+      line: [
+        "17 Austhorpe Road",
+        "Crossgates",
+        "Leeds"
+      ],
+      postalCode: "LS15 8BA"
+    }
+  ],
+  active: true,
+  type: [
+    {
+      coding: [
+        {
+          system: "https://fhir.nhs.uk/CodeSystem/organisation-role",
+          code: "182",
+          display: "PHARMACY"
+        }
+      ]
+    }
+  ],
+  name: "The Simple Pharmacy",
+  telecom: [
+    {
+      system: "phone",
+      use: "work",
+      value: "0113 3180277"
+    }
+  ]
+}
+
+const containedPractitionerRole: fhir.PractitionerRole = {
```

Comment:
should be able to delete both this and the above `const`s when pulling from the dispense

Reply:
done